x-pack/packages/kbn-elastic-assistant/impl/assistant/prompt_editor/system_prompt/select_system_prompt/index.tsx

```diff
@@ -174,20 +188,55 @@ const SelectSystemPromptComponent: React.FC<Props> = ({
               onBlur={handleOnBlur}
               options={[...options, addNewSystemPrompt]}
               placeholder={i18n.SELECT_A_SYSTEM_PROMPT}
-              valueOfSelected={selectedPrompt?.id ?? allSystemPrompts[0]?.id}
+              valueOfSelected={valueOfSelected}
+              prepend={isFlyoutMode ? 'Select prompt' : undefined}
```

Comment:
```suggestion
              prepend={isFlyoutMode && !isSettingsModalVisible ? 'Select prompt' : undefined}
```

Need to add a settings check here otherwise the prepend will show in settings:


<p align="center">
  <img width="500" src="https://github.com/elastic/kibana/assets/2946766/8a26f6ec-f2ba-4c06-9e65-f4d3d73a809d" />
</p>